### PR TITLE
Vmimage automatic documentation

### PR DESCRIPTION
**Pull request by richtja**
This generates a documentation table with every tested distribution for
vmimage.

Reference: #3937
Signed-off-by: Jan Richter <jarichte@redhat.com>

### CODE REVIEW

**Comment by clebergnu** [comment]
@richtja shouldn't the `docs/source/guides/writer/libs/vmimage.rst.data/supported_images.csv` file be removed from version control and only created/used at docs creation time?

File: docs/source/conf.py [hunk 1]
```diff
@@ -42,7 +42,37 @@ def generate_reference():
         reference.write(result.stdout_text)
 
 
+def generate_vmimage_distro():
+    from avocado.core import parameters, varianter
```
Comment (beraldoleal) [inline docs/source/conf.py:46]: any reason to not adding this to the top of the file?

File: docs/source/guides/writer/libs/vmimage.rst.data/supported_images.csv [hunk 2]
```diff
@@ -0,0 +1,50 @@
+Provider, Version, Architecture
```
Comment (beraldoleal) [inline docs/source/guides/writer/libs/vmimage.rst.data/supported_images.csv:1]: This is going to be generated automatically, right? Maybe we could add this to the .gitignore file.

**Comment by richtja** [comment]
@clebergnu, @beraldoleal, @willianrampazzo, Thanks for your comments I will create v2.

**Comment by richtja** [comment]
Closing in favor #4090

**Pull request closed without merging**